local-testing-support/ubuntu-20.04/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+# Start a bash session with this container with this command:
+# docker-compose run --rm ubuntu /bin/bash
+#
+# After opening a bash shell in the container, test Salt configuration of an
+# endpoint with this command:
+# salt-call --local state.highstate --file-root=/srv/salt --pillar-root=/srv/pillar --id=endpoint
+
+version: "3"
+services:
+  ubuntu:
+    image: ubuntu-20.04-salt-3001:latest
+    volumes:
+      # Set these paths to your salt and pillar folders.
+      - /Users/giontc1/Documents/MICrONS/boss-manage/salt_stack/salt:/srv/salt
+      - /Users/giontc1/Documents/MICrONS/boss-manage/salt_stack/pillar:/srv/pillar
```

Comment:
hard coded.  If this is just for testing it is probably ok.

Reply:
Yes user should replace with the location of their Salt files. This config file is provided for convenience and as an example. Without it, you would be providing all these paths to Docker from the command line.